### PR TITLE
[IMP]10708 add module mass_items

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

File: mass_items/__init__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline mass_items/__init__.py]: Headers + README
Propose to the OCA
Change the name: product_mass_selection?

Comment (elicoidal) [inline mass_items/__init__.py]: In theory you should have one module for sales, one for purchase, one for SM

Comment (elicoidal) [inline mass_items/__init__.py]: then sale_mass_selection, purchase_mass_selection, stock_mass_selection

File: mass_items/README.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
+'description': """
```
Comment (elicoidal) [inline mass_items/README.rst]: Use newest README template

**Review by elicoidal — CHANGES_REQUESTED** [review]
mostly README

**Comment by noahzaozao** [comment]
@elicoidal please review this pr again

**Comment by elicoidal** [comment]
Change the name of the module to mass_item_selection then merge

**Pull request merged**